controllers/components/datasciencepipelines/datasciencepipelines.go

```diff
@@ -26,11 +26,19 @@ func init() { //nolint:gochecknoinits
 }
 
 func (s *componentHandler) GetName() string {
-	return componentsv1.DashboardComponentName
+	return componentsv1.DataSciencePipelinesComponentName
 }
 
-func (s *componentHandler) GetManagementState(dsc *dscv1.DataScienceCluster) operatorv1.ManagementState {
-	return dsc.Spec.Components.Dashboard.ManagementState
+func (s *componentHandler) GetManagementState(dsc *dscv1.DataScienceCluster) string {
+	if s == nil {
+		return "Removed"
```

Comment:
I would do `return string(operatorv1.Removed)` so it is always consistent with the constant

Reply:
i first made the one with return string(operatorv1.Removed)
then i thought to convert a const to string, is that extra step to just return a string.
but yeah, i get your point, i can get that reverted